my-portfolio/src/components/Contact.js

```diff
@@ -0,0 +1,142 @@
+import React, { Component } from "react";
+import { Container, Row, Col, Table } from "react-bootstrap";
+import logo4 from "../7.png";
```

Comment:
It is hard to know what exactly 7.png is . Same with the rest of images (1.png, 2.png ...)